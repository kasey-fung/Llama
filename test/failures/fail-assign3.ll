int main():
    string x
    int y
    bool z

    x = true
    z = 'a'
    z = 100
    y = 200
