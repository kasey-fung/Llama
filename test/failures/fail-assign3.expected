Fatal error: exception Failure("illegal assignment bool = int in z = 100")